graphql/subcourse/fields.ts

```diff
@@ -137,11 +137,15 @@ export class ExtendedFieldsSubcourseResolver {
     @LimitedQuery()
     async subcourseSearch(
         @Arg('search') search: string,
+        @Arg('courseStates', () => [String], { nullable: true }) courseStates: course_coursestate_enum[],
```

Comment:
What happens when you omit the courseStates?

Reply:
It just fetches all the subcourses as before, ofc I tried that out. :)